my_code.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import matplotlib
+import os
+from astropy.io import fits
+import glob
+data = fits.getdata('/Users/bostroem/Desktop/images/hubble_img.fits')
```

Comment:
The file name is hard coded in the program. Isn't better to ask the user to fill the file name?